src/sieve/sieve_Gauss.cpp

```diff
@@ -0,0 +1,323 @@
+/*
```

Comment:
done, fixed above. It seems compilting now with
make latsieve
there was some change in the codebase I guess which made it broken :)